### PR TITLE
Fix empty container layer history and UI

### LINKED ISSUE #21248: Table UI is overflow when package label is too long
### Description

Hi all, I found a UI problem when I try to publish the container image to Gitea Package by Paketo + Cloud Native Build Pack:
- Only divider in the "Layers" section if the layer name is empty
  - Maybe need a default shown value such as `(Empty)`
- The content is overflow in the "Labels" section if the label is too long
  - Maybe we can show the split content in the table, and provide a popover view to show the full content of the label.

### Screenshots

image

### Gitea Version

1.17.2

### Can you reproduce the bug on the Gitea demo site?

Yes

### Operating System

Windows 11

### Browser Version

Microsoft Edge 105.0.1343.42

### PR DESCRIPTION
**Pull request by KN4CK3R**
Fixes #21248

before
grafik

after
grafik

### CODE REVIEW

**Comment by siarie** [comment]
I think it can be solved with adding `word-break: break-word;` to the `td` element. however the key column may wrapped as well. This can be solved by adding fixed size on the key column.

Another solution is to wrap the value inside the `pre` tag. here the example:
```html
<!-- CSS -->
<style>
.ui.table td pre {
  white-space: pre-wrap;
  word-break: break-word;
}
</style>

<!-- HTML -->
<table>
  ...
  <tbody>
    <tr>
      <td>the.key</td>
      <td><pre>the value here</pre></td>
    </tr>
  </tbody>
</table>
```

**Comment by silverwind** [comment]
Table overflow has it's own weird rules that generally don't follow what other HTML elements do. `table-layout: fixed` would achieve wrapping, but the page gets pretty long with with all that JSON. Maybe it's better to wrap the JSON into a `<pre><code>` and put a `overflow-x: auto` for horizontal scrolling.

File: templates/package/content/container.tmpl [hunk 1]
```diff
@@ -47,7 +47,7 @@
 	{{if .PackageDescriptor.Metadata.ImageLayers}}
 		<h4 class="ui top attached header">{{.locale.Tr "packages.container.layers"}}</h4>
 		<div class="ui attached segment">
-			<table id="notice-table" class="ui very basic compact table">
+			<table class="ui very basic compact table">
```
Comment (silverwind) [inline templates/package/content/container.tmpl:50]:
```suggestion
			<table class="ui very basic compact fixed table">
```

File: templates/package/content/container.tmpl [hunk 2]
```diff
@@ -61,7 +61,7 @@
 	{{if .PackageDescriptor.Metadata.Labels}}
 		<h4 class="ui top attached header">{{.locale.Tr "packages.container.labels"}}</h4>
 		<div class="ui attached segment">
-			<table id="notice-table" class="ui very basic compact table">
+			<table class="ui very basic compact table">
```
Comment (silverwind) [inline templates/package/content/container.tmpl:64]:
```suggestion
			<table class="ui very basic compact fixed table">
```

**Comment by silverwind** [comment]
You can add the `fixed` class to `table` to fix it. It makes both columns 50% width which is often too wide, but I can't really find another way to fix this, besides swapping table for flexbox which could do what we want.

File: web_src/less/_package.less [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.container-labels td:nth-child(2) {
+  word-break: break-word;
```
Comment (silverwind) [inline web_src/less/_package.less]:
```suggestion
  overflow-wrap: break-word;
```

`word-break: break-word` is deprecated [as per MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break#break-word). Please test it, I'm not totally use this is equivalent.

**Comment by lafriks** [comment]
Would be nice if key would be vertically aligned to top

File: web_src/less/_package.less (hunk 3 above)
Comment (KN4CK3R) [reply web_src/less/_package.less]: `overflow-wrap: anywhere;` works, thanks!

Comment (silverwind) [reply web_src/less/_package.less]: Ah yes, it needs `anywhere` because the string don't contain whitespace.

**Review by silverwind — APPROVED** [review]

**Review by 6543 — APPROVED** [review]

**Pull request merged**